ingress-nginx-controller-1.12.yaml

```diff
@@ -498,6 +502,64 @@ pipeline:
         && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/usr/bin/nginx
 
 subpackages:
+  - name: ingress-nginx-controller-bitnami-compat-${{vars.nginx-ingress-major-minor}}
+    description: Compatibility package with bitnami/nginx-ingress-controller image
+    dependencies:
+      provides:
+        - ingress-nginx-controller-bitnami-compat=${{package.full-version}}
+      runtime:
+        - bash # https://github.com/bitnami/charts/blob/db53e622ecf1be5a78d1ce683cec8baa41110fa3/bitnami/nginx-ingress-controller/templates/controller-deployment.yaml#L89
+        - gmp
+        - gnutls
+        - ingress-nginx-controller-compat-${{vars.nginx-ingress-major-minor}}
+        - libffi
+        #- libhogweed6 missing pkg/so
+        - libmaxminddb-dev
+        - librtmp
+        - libssh
+        - libtasn1
+        - nettle
+        - p11-kit
+        - procps
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: nginx-ingress-controller
+          version-path: ${{vars.nginx-ingress-major}}/debian-12
+      - name: Ensure symlinks
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/common/bin
+          ln -s /usr/bin/mmdblookup ${{targets.subpkgdir}}/opt/bitnami/common/bin/mmdblookup
+          ln -s /usr/lib ${{targets.subpkgdir}}/opt/bitnami/common/lib
```

Comment:
maybe `ln -s /usr/local/bin/nginx /usr/bin/nginx` as well because in our package we've it at `/usr/local/bin/nginx` but upstream has it at `/usr/bin/nginx`

Reply:
That seems to be the opposite @kranurag7

Reply:
Bitnami:
```
ls -l /usr/bin/nginx
lrwxrwxrwx. 1 root root 27 Feb 20 14:36 /usr/bin/nginx -> /usr/local/nginx/sbin/nginx
```

Chainguard ingress controller non-Bitnami:
```
 ls -l /usr/local/bin/nginx
lrwxrwxrwx    1 root     root            14 Feb 20 15:47 /usr/local/bin/nginx -> /usr/bin/nginx
```

Reply:
Hmm, what about `/usr/local/nginx/sbin/nginx` then? Should that also be symlinked?

Reply:
@wojciechka /usr/local/nginx/sbin/nginx in Bitnami one is the source file. You think we should provide a symlink also for that like `ln -s /usr/bin/nginx /usr/local/nginx/sbin/nginx`?

Reply:
+1, symlink this as well

I'm wondering if that should to go bitnami or upstream compat? Can you check whether upstream nginx image has this file as well? If yes, add to compat. If not, add to bitnami-compat.